UtopiaCity/Common/DbInitializer.cs

```diff
@@ -17,6 +17,8 @@ public static void RegisterSubInitializers()
             // simple sets
             subDbInitializers.Add(new EmergencyReportInitializer());
 
```

Comment:
I think they could be without empty lines, just block of code - block of simple initializers etc.